### PR TITLE
Fix flakiness in E2E tests

### PR DESCRIPTION
**Pull request by banool**
## Summary
A few changes in one PR. I'll rebase rather than squash:
- **Increase wait time for local testnet startup**
- **Hit / in api_token for full node requests test, log on error**
  - Log to help debug why it failed.
- **Use decimals function rather than is_registered in function generics test**

## Test Plan
See that CI passes, hopefully with fewer retries than normally required.

### CODE REVIEW

File: tests/e2e/api/general.test.ts [hunk 1]
```diff
@@ -174,13 +174,13 @@ describe("general api", () => {
...
 
       const payload2: InputViewFunctionData = {
-        function: "0x1::coin::is_account_registered",
+        function: "0x1::coin::decimals",
         typeArguments: ["0x1::aptos_coin::AptosCoin"],
-        functionArguments: ["0x1"],
+        functionArguments: [],
       };
 
-      const isRegistered = (await aptos.view<[boolean]>({ payload: payload2 }))[0];
```
Comment (0xmaayan) [inline tests/e2e/api/general.test.ts:182]: have we figured out why it returns true instead of false? might be a higher level problem?

**Review by 0xmaayan — APPROVED** [review]

File: tests/e2e/api/general.test.ts (hunk 1 above)
Comment (banool) [reply tests/e2e/api/general.test.ts:182]: Nope, good point though, we can investigate separately.

Comment (banool) [reply tests/e2e/api/general.test.ts:182]: I'm guessing some other test transfers coin to that account, but bc the test order is not deterministic the result is different sometimes.

**Comment by banool** [comment]
Bingo, tests passed on the first attempt.

**Pull request merged**